docs/client/basic/toc.js

```diff
@@ -48,7 +48,7 @@ var sections = [
       item("Defining templates in HTML", {id: "defining-templates"}),
       item("Template.<em>name</em>.helpers", {longname: "Template#helpers"}),
       item("Template.<em>name</em>.events", {longname: "Template#events"}),
-      item("Template.<em>name</em>.rendered", {longname: "Template#rendered"}),
+      item("Template.<em>name</em>.onRendered", {longname: "Template#onRendered"}),
```

Comment:
Haha the previous comment was a joke :] it's the only answer, so we will just edit it.